src/mcp_analyzer/checkers/token_efficiency.py

```
@@ -398,6 +405,23 @@ async def _measure_response_sizes(
+                        console.print(
+                            f"\n[dim]Tool[/dim] [cyan]{tool_name}[/cyan] "
+                            f"[dim]Scenario[/dim] [magenta]{scenario.name}[/magenta] "
+                            f"[dim](~{token_count} tokens)[/dim]"
+                        )
+                        # Prefer pretty JSON when possible
+                        if isinstance(response, (dict, list)):
+                            console.print_json(data=response, default=str)
+                        else:
+                            # Fallback: stringify non-JSON responses
+                            console.print(str(response))
+                    except Exception as print_exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)